packages/gnome-shell/src/misc/dateUtils.d.ts

```diff
@@ -0,0 +1,37 @@
+import GLib from '@girs/glib-2.0';
+
+/**
+ * Formats a Date object according to a C sprintf-style string using
+ * the cached local timezone.
+ *
+ * @see https://gitlab.gnome.org/GNOME/gnome-shell/-/blob/main/js/misc/dateUtils.js#L41
+ *
+ * @param {Date} date a Date object
+ * @param {string} format a format String for the date
+ * @returns {string}
```

Comment:
Please add a `@version 49` tag to every doc comment here, to clarify that the types were checked against Gnome 49.  

This helps us keep track of the state of types across Gnome releases.